gno.land/cmd/gnoland/main.go

```diff
@@ -169,6 +207,30 @@ func exec(c *gnolandCfg) error {
 	select {} // run forever
 }
 
+// getIndexerConfig constructs an indexer config from provided user options
+func getIndexerConfig(c *gnolandCfg) (*indexercfg.Config, error) {
```

Comment:
can indexercfg.Config just be embedded in gnolandCfg? 
why not indexercfg -> types? the intent of "types" in the project is to collect these exposed types, where a whole package isn't necessary.

Reply:
We pass the ~Indexer~ EventStore configuration around from this entry point in the main binary, down to the internal configs (although, within the now `tm2` package; this PR was started before the cleanup by @moul). I guess we can move it out to the `types` package in `bft` (I assume this is what you mean when you say `types`), the original reasoning behind keeping it in the `eventstore` package is to have it be contained logic-wise.

Do you still think we should move it out to `bft/types`?